plugins/inputs/snmp_trap/snmp_trap.go

```diff
@@ -34,7 +34,6 @@ type SnmpTrap struct {
 	Version        string            `toml:"version"`
 
 	// Settings for version 3
-	ContextName string `toml:"context_name"`
```

Comment:
Usually we would leave the option, but deprecate it.  Since I expect the option isn't in wide use I suggest we remove the documentation for it too.

https://github.com/influxdata/telegraf/wiki/Deprecation#deprecate-options

Reply:
Nevermind, I see this is just added to master recently.